template/README.md

```diff
@@ -30,6 +30,7 @@ This app has been generated using [react-native-template-redbeard](https://githu
 - `localization/` - Things related to user locale
 - `navigation/` - Navigators, routes
 - `redux/` - Actions, reducers, sagas
+- `remote/` - Remote state (via TanStack Query)
```

Comment:
This indicates remote state (server state) in terms of reflecting inside the app what's stored on some remote server (backend). It's keeping networking state - loading, errors and data from requests and responses.

It's different than `redux/`, which is client local state, that doesn't need to by synchronised with server. It's updated by local user's action or some effects from remote state updates.

This structure allows to keep them separate.